src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1398,7 +1398,7 @@ public static bool IsGlobalStatementContext(this SyntaxTree syntaxTree, int posi
             return token.IsBeginningOfGlobalStatementContext();
         }
 
-        public static bool IsInstanceContext(this SyntaxTree syntaxTree, int position, SyntaxToken tokenOnLeftOfPosition, CancellationToken cancellationToken)
+        public static bool IsInstanceContext(this SyntaxTree syntaxTree, int position, SyntaxToken tokenOnLeftOfPosition, bool isAnyExpressionContext, CancellationToken cancellationToken)
```

Comment:
Why does this have the "isAnyExpressionContext" parameter? If you detect that you're in a (not static) expression bodied member, it's automatically an expression, right?